src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -9,11 +9,11 @@
                                  :value="ds.selections.datasetId"
                                  @update="updateDataSource(ds.config.id, $event)">
                     </data-source>
-                    <filters v-if="ds.config.showFilters && ds.filters && ds.selections.selectedFilterOptions"
+                    <filters-comp v-if="ds.config.showFilters && ds.filters && ds.selections.selectedFilterOptions"
```

Comment:
is there a reason why renamed this component ?

Reply:
is it because you get a warning now with a single-word component name?

Reply:
actually because there is a computed value called filters too and vue seems to be getting confused between the two :laughing:

Reply:
is it not better to rather rename the computed property ?